rotkehlchen/utils/misc.py

```diff
@@ -376,11 +376,12 @@ def is_valid_ethereum_tx_hash(val: str) -> bool:
 def create_order_by_rules_list(
         data: Dict[str, Any],
         default_order_by_field: str = 'timestamp',
+        is_ascending: bool = False,
 ) -> List[Tuple[str, bool]]:
     """Create a list of attributes and sorting order taking values from DBOrderBySchema
-    to be used by the filters that allow sorting. By default the attribute used for sorting is
+    to be used by the filters that allow sorting. By default, the attribute used for sorting is
     timestamp and the ascending value for this field is False.
```

Comment:
`the ascending value for this field is False.` I guess needs to change

Reply:
The default value is still the value stored by the variable `is_ascending` which is False. Do you get?

Reply:
Yeah, I get it. Perhaps can leave it as is, but I would still mention is_ascending in the docstring